Formula/docker-compose.rb

```diff
@@ -19,7 +19,6 @@ class DockerCompose < Formula
   depends_on "go" => :build
 
   def install
-    ENV["CGO_ENABLED"] = "0"
```

Comment:
Without this, the build succeeded locally.

Reply:
Can confirm that this is a legit fix: https://github.com/docker/compose/commit/7a42ba7eec0569d188312543882d9c7ede5c1855

Reply:
👍